src/coreComponents/physicsSolvers/fluidFlow/CompositionalMultiphaseFVM.cpp

```diff
@@ -361,16 +361,18 @@ real64 CompositionalMultiphaseFVM::scalingForSystemSolution( DomainPartition con
 
   string const dofKey = dofManager.getKey( viewKeyStruct::elemDofFieldString() );
   real64 scalingFactor = 1.0;
+  real64 maxDeltaPres = 0.0, maxDeltaCompDens = 0.0, maxDeltaTemp = 0.0;
```

Comment:
I can't quite tell if the locally calculated pressure and temperature changes are absolute. If not I would expect `maxDeltaPres` and `maxDeltaTemp` to start as large negative values.

Reply:
```
    // compute the change in pressure
    real64 const pres = m_pressure[ei];
    real64 const absPresChange = LvArray::math::abs( m_localSolution[stack.localRow] );
    if( stack.localMaxDeltaPres < absPresChange )
    {
      stack.localMaxDeltaPres = absPresChange;
    }
```
this is how the change is computed, so it should be absolute